grpc-proto/proto/query.proto

```diff
@@ -57,9 +57,11 @@ message UdtValue {
 
 // A UUID value (for CQL types uuid and timeuuid).
 message Uuid {
-  // TODO: Is this going to have endian problems representing it this way?
-  fixed64 msb = 1;
-  fixed64 lsb = 2;
+  bytes value = 1;
```

Comment:
Does protobuf have a way to indicate byte sequence of fixed length? I should remember this but I don't -- I assume there isn't but thought to ask just in case; we'd want to verify it is 16 bytes if it was possible.

Reply:
Good thought. Unfortunately, no. I wish it was possible though.